tests/contentproviders/test_dataverse.py

```diff
@@ -0,0 +1,67 @@
+import hashlib
+import os
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from repo2docker.contentproviders import Dataverse
+
+test_dv = Dataverse()
+harvard_dv = next(_ for _ in test_dv.hosts if _["name"] == "Harvard Dataverse")
```

Comment:
Can you add a comment explaining what this iterates through, and what we end up comparing in the test? Are the names `Harvard Dataverse`/`CIMMYT Research Data` considered immutable properties of repository?

Reply:
@manics I'll do so, although I didn't add these - they were already there in the existing file.

Reply:
@manics I've incorporated this into https://github.com/jupyterhub/repo2docker/pull/1390, and simply removed using "host" this way completely.